libs/nx-plugin/src/executors/code-gen/schema.json

```diff
@@ -33,15 +33,15 @@
 
         "subFolder": {
             "type": "string",
-            "description": "Sub-folder to add the generated code to.",
+            "description": "The Sub-folder of the project to add the generated code to.",
             "default": "./"
         },
 
         "dryRun": {
             "type": "boolean",
-            "description": "Only show what would be generated.",
+            "description": "Only show what would be generated don't modify files.",
```

Comment:
I think this needs some punctuation here?

Reply:
Fixed